src/backend/access/transam/xact.c

```diff
@@ -95,6 +95,9 @@
 #include "utils/vmem_tracker.h"
 #include "cdb/cdbdisp.h"
 #include "postmaster/autovacuum.h"
+#include "utils/combocid.h"
+
+combocid_hook_type combocid_hook = NULL;
```

Comment:
We need to change the name from combocid to CCI_hook?

Reply:
OK, I'll change the hook name later